test/runnable/cppa.d

```diff
@@ -3,6 +3,9 @@
 // EXTRA_FILES: extra-files/cppb.h
 // CXXFLAGS(linux freebsd osx netbsd dragonflybsd): -std=c++11
 
+// Filter a spurious warning on Semaphore:
+// TRANSFORM_OUTPUT: remove_lines("warning: relocation refers to discarded section")
+
```

Comment:
This is not spurious, and as of Ubuntu 22.04/gcc-12 this linker warning is now an error.